pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/LLCRealtimeClusterSplitCommitEndWithMetadataTest.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.pinot.integration.tests;
+
+import org.apache.commons.configuration.Configuration;
+import org.apache.pinot.common.utils.CommonConstants;
+import org.apache.pinot.controller.ControllerConf;
+
+/**
+ * Integration test that extends LLCRealtimeClusterIntegrationTest but with split commit enabled and segment commit end
+ * with metadata file upload enabled.
+ */
+public class LLCRealtimeClusterSplitCommitEndWithMetadataTest extends LLCRealtimeClusterIntegrationTest {
```

Comment:
Instead of adding a new test (that increases test time),  we can just change the split commit test to just use the new controller endpoint to commit with metadata. We should deprecate the one that commits without metadata, since this one is far superior. The only reason to use the other one is for backward compat for one release.